data-source/json/test_answer_action_redirect_to_list_add_question.json

```diff
@@ -0,0 +1,217 @@
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "sections": [
+        {
+            "id": "section",
+            "groups": [
+                {
+                    "id": "group",
+                    "title": "List",
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "anyone-else-live-here-block",
+                            "question": {
+                                "type": "General",
+                                "id": "anyone-else-live-here-question",
+                                "title": "Does anyone usually live at 1 Pleasant Lane?",
+                                "answers": [
+                                    {
+                                        "type": "Radio",
+                                        "id": "anyone-else-live-here-answer",
+                                        "mandatory": true,
+                                        "options": [
+                                            {
+                                                "label": "Yes",
+                                                "value": "Yes",
+                                                "action": {
+                                                    "type": "RedirectToListAddQuestion",
+                                                    "params": {
+                                                        "block_id": "add-person",
+                                                        "list_name": "people"
+                                                    }
+                                                }
+                                            },
+                                            {
+                                                "label": "No",
+                                                "value": "No"
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "id": "list-collector",
+                            "type": "ListCollector",
+                            "for_list": "people",
+                            "add_answer": {
+                                "id": "anyone-else",
+                                "value": "Yes"
+                            },
+                            "remove_answer": {
+                                "id": "remove-confirmation",
+                                "value": "Yes"
+                            },
+                            "question": {
+                                "id": "confirmation-question",
+                                "type": "General",
+                                "title": "Does anyone else live here?",
```

Comment:
It would help with the understanding of the schema if this was phrased as in the previous question i.e. "Does anyone else live at 1 Pleasant Lane?"

Reply:
Done.